### ISSUE TITLE
CRAN release?

### ISSUE THREAD

**Issue by privefl** [PheWAS/PheWAS#48]
Thanks for this great package.
Do you plan to release it on CRAN?
Do you want some help doing it?

**Comment by skanoni** [comment]
Hi,
I am also trying to import the PheWAS library in a safe haven environment (no devtools connection). Is there any way around it?

**Comment by RobertJCarroll** [comment]
Hi all. I think a CRAN release would be great, but I haven't looked into it recently (ie, how much work it'd take to pass the checks from where it is now).
Re getting around it- one can build the package on another machine then send that file over. The package itself is all native R (though uses libraries that are not), so it should be pretty straight forward.

**Comment by privefl** [comment]
I can have a quick look tomorrow if you want.

**Comment by skanoni** [comment]
That would be awesome!
________________________________

**Comment by privefl** [comment]
@RobertJCarroll  Would you be okay using roxygen2 for writing documentation of functions?

**Comment by RobertJCarroll** [comment]
I haven't used roxygen in a long time- is there a benefit to transitioning over the existing Rd files?

**Comment by privefl** [comment]
Easier to maintain and modify.

**Comment by eliasallara** [comment]
Thanks for the great package! May I say +1 for CRAN release - we are also trying to use it in a safe haven environment that does not have a devtools connection. As safe havens are becoming more common, a CRAN release will likely become useful to a lot of people.

**Comment by pettyalex** [comment]
@RobertJCarroll I see some new development efforts on the "2.0" branch, any chance that this can get submitted to CRAN with the next release? It'd make it a lot easier to install in a wide variety of environments.

I took a look at what it would take to clean the current master branch up for CRAN, and it's not that many things. I'd submit a PR with it, but I'm not sure if master is the current development branch or if it's the 2.0.

**Comment by RobertJCarroll** [comment]
Yes, @adlewismbb is refreshing the codebase and preparing the package for CRAN now in the 2.0 branch. Much of the work is around making sure it's "stable" for long term in cleaning up the function names and parameters as CRAN doesn't like breaking changes.

**Comment by privefl** [comment]
Any update on this? :-)

**Comment by pettyalex** [comment]
@RobertJCarroll Is there anything we can do help this along, or would it be possible to update the existing phewas package to be more easily installable from CRAN? I keep hearing from researchers that it's more difficult than expected to install this package in various environments.

**Comment by pettyalex** [comment]
Checking on this 1.5 years later: Which branch is the development branch? My group is regularly running up against memory limits and I'd love to see if there's any tweaks I can do to help, but I'm not sure if the 2.0 release has happened yet or not.
